modules/system/models/MailLayout.php

```diff
@@ -134,4 +141,19 @@ protected static function getTemplateSections($code)
     {
         return MailParser::parse(FileHelper::get(View::make($code)->getPath()));
     }
+
+    public function getOptionsAttribute($value)
```

Comment:
why not seed/migrate that default instead checking in an mutator?
(I know that mysql is broken with column text and setting defaults, but a simple 
```php
DB::table('system_mail_layouts')->update(['options'` => '{inline_css: 1}'])
```
ether in the migration file, or in a separate seed file, would do the trick)

Reply:
@munxar does that work for all DB drivers that October supports? It seems like it would for MySQL but I'm not sure about the others

Reply:
changed it now, removed a lot of lines.